docs/modules/ROOT/pages/backup-and-restore.adoc

```diff
@@ -23,7 +23,7 @@ You can keep at most 10 backups. The oldest backup will be deleted automatically
 
 == Restoring a New Cluster from a Backup
 
-When you have at least one backup, you can use it to bootstrap a new cluster.
+When you have at least one backup, you can use it to bootstrap a new cluster. The new cluster is created using the current Hazelcast version. To see the Hazelcast version in use, either look at the xref:release-notes.adoc[release notes] or go to the dashboard of an existing cluster and view the *Cluster Details*.
```

Comment:
> or go to the dashboard of an existing cluster and view the *Cluster Details*.

That does not guarantee that new clusters will run the same Hazelcast version (since we do not update versions automatically -- they are not always backward compatible)

Reply:
Text updated to resolve comments.